src/test/java/com/intrafind/log4shell/Log4shellUpdateTest.java

```diff
@@ -79,41 +85,59 @@ public void before() throws IOException {
     deepFatJar.write(99);
     deepFatJar.closeEntry();
     deepFatJar.close();
+    Files.createFile(service.resolve("log4j.properties"));
+    final Path serviceLib = Files.createDirectory(service.resolve("lib"));
+    Files.createFile(serviceLib.resolve("log4j-1.2.17.jar"));
+    final Path serviceConf = Files.createDirectory(service.resolve("conf"));
+    Files.createFile(serviceConf.resolve("wrapper.conf"));
     final Path serviceSubOld = Files.createDirectory(service.resolve("subOld"));
     Files.createFile(serviceSubOld.resolve("log4j-1.2-api-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-api-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-core-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-jcl-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-layout-template-json-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-slf4j-impl-2.11.0.jar"));
-    Files.createFile(serviceSubOld.resolve("log4j-1.2.17.jar"));
     final Path serviceSubNew = Files.createDirectory(service.resolve("subNew"));
-    Files.createFile(serviceSubNew.resolve("log4j-core-2.16.0.jar"));
+    Files.createFile(serviceSubNew.resolve("log4j-core-2.17.1.jar"));
     final Path serviceSubNewer = Files.createDirectory(service.resolve("subNewer"));
     Files.createFile(serviceSubNewer.resolve("log4j-core-2.30.0.jar"));
     final Path sub17 = Files.createDirectory(service.resolve("sub17"));
     Files.createFile(sub17.resolve("log4j-core-2.17.0.jar"));
+    final Path app = Files.createDirectory(tempDir.resolve("app"));
+    final Path appLib = Files.createDirectory(app.resolve("lib"));
+    Files.createFile(appLib.resolve("log4j-1.2.17.jar"));
+    Files.createFile(appLib.resolve("slf4j-log4j12-1.7.0.jar"));
+    final Path appConf = Files.createDirectory(app.resolve("conf"));
+    Files.createFile(appConf.resolve("log4j.properties"));
+    final Path appBat = Files.createDirectory(app.resolve("bat"));
```

Comment:
shouldn't we check windows & linux, or do we just support windows clearly documented in the help of the program?

Reply:
I'll add a Linux test